cmd/juju/interact/pollster.go

```diff
@@ -393,7 +394,11 @@ func (p *Pollster) queryOneSchema(schema *jsonschema.Schema) (interface{}, error
 		// anything
 		a, err = p.Enter(schema.Singular)
 	case jsonschema.FormatURI:
```

Comment:
It seems like this could get out of control. Is there a more general way for verifying the schema? Maybe a verification lambda should be part of the construction of a `Pollster`?